library/src/main/java/com/codetroopers/betterpickers/radialtimepicker/RadialTimePickerDialog.java

```diff
@@ -516,7 +516,7 @@ private void setCurrentItemShowing(int index, boolean animateCircle, boolean del
      * @return true if the key was successfully processed, false otherwise.
      */
     private boolean processKeyUp(int keyCode) {
-        if (keyCode == KeyEvent.KEYCODE_ESCAPE || keyCode == KeyEvent.KEYCODE_BACK) {
```

Comment:
Otherwise there is no way to distinguish if the dialog was cancelled by user or dismissed (i.e. on screen rotation). By removing the check, onCancel() method is called for both tapping outside of the dialog and pressing the back button (i.e. the user decides to cancel the dialog). It allowed me to differentiate between the dialog being dismissed on rotation and cancelled by user.